docs/proposals/formatted-output.md

```diff
@@ -334,13 +334,13 @@ $REGISTRY/$REPO@sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4e
 
 View an artifact's referrers manifest in pretty JSON output. The following fields should be outputted:
```

Comment:
After #1403 is completed, the subject manifest will also be in the output. This should be reflected in this spec. I have named the struct field temporarily as `subject`, we can discuss about it.

Reply:
We will keep what this spec proposed per our discussion and won't introduce an additional `subject` field since it might confuse users.